doc/source/wsrep-system-index.rst

```diff
@@ -329,16 +329,27 @@ This variable can be used to reject queries for the node. This can be useful dur
 
 .. variable:: wsrep_replicate_myisam
 
-   :version 5.6.24-25.11: Variable available both in global ans session scope
+   :version 5.6.24-25.11: Variable available both in global and session scope
    :cli: Yes
    :conf: Yes
    :scope: Session, Global
    :dyn: No
    :default: OFF
 
-This variable controls if *MyISAM* will be replicated or not. *MyISAM* replication is still experimental and that is one of the reasons why this variable is set to ``OFF`` by default. *MyISAM* ``CREATE TABLE`` isn't replicated any more when :variable:`wsrep_replicate_myisam` is ``OFF``.
+This variable defines whether MyISAM should be replicated or not. It is disabled by default, because MyISAM replication is still experimental. MyISAM ``CREATE TABLE`` isn't replicated any more when :variable:`wsrep_replicate_myisam` is disabled.
```

Comment:
MyISAM CREATE TABLE isn't replicated ... 
This has changed starting 5.6.27. Please check updated semantics. We replicate DDL but block DML based on wsrep_replicate_myisam configuration.